docs/install.md

```diff
@@ -690,6 +690,10 @@ A login prompt will appear. Use the initial credentials:
     ssh root@dietpi
     ```
 
+??? info "Option: Automatic base installation at first boot (running an _unattended base installation_)"
+
+    DietPi offers the option for an automatic first boot installation. See section ["How to do an automatic base installation at first boot"](../usage/#how-to-do-an-automatic-base-installation-at-first-boot) for details.
+
```

Comment:
Let's move that to a place where the image has not been bootet yet (and it is hence too late 😉). There is the WiFi config block already in most install tabs, probably this should go there just below the WiFi block. To avoid doubled Markdown code, we can enable MkDocs includes. I wanted to do that for the `dietpi-software` block at the top of each category page already. But I can do that both in a separate PR.

Reply:
Ok. Moved it upwards and also copied it to the BIOS direct image section.